specs/Environment/AirQualityObserved/doc/spec.md

```diff
@@ -133,8 +133,112 @@ Below is the description of the attribute to be used for option A/.
 **Note**: JSON Schemas only capture the NGSI simplified representation, this means that to test the JSON schema examples with
 a [FIWARE NGSI version 2](http://fiware.github.io/specifications/ngsiv2/stable) API implementation, you need to use the `keyValues`
 mode (`options=keyValues`).
+
+## Examples of use 1 (Normalized Format)
+
+```json
+{
+    "id": "Madrid-AmbientObserved-28079004-2016-03-15T11:00:00",
+    "type": "AirQualityObserved",
+    "dateObserved": {
+        "value": "2016-03-15T11:00:00/2016-03-15T12:00:00"
```

Comment:
will the context broker return this or also the type?

Reply:
we are conveying here just the Entity Representation that can be used by Context Producers to create the corresponding Entity.